contracts/dynamic-callee-contract/.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2.1
```

Comment:
is this file needed?

Reply:
It was created using a template. I removed CI all. thks!